commonknowledge/website
# Security upgrades

This change pins specific versions of dependencies as resolutions in package.json file to fix security vulnerabilities. 

## How Can It Be Tested?
Download the branch, run yarn to install dependencies, and yarn audit to see vulnerabilities. 

## Types of changes


## Checklist:
